packages/neos-ui-redux-store/src/User/Settings/index.ts

```diff
@@ -43,10 +47,14 @@ export const reducer = (state: State = defaultState, action: Action | InitAction
     switch (action.type) {
         case system.INIT:
             draft.isAutoPublishingEnabled = action.payload.user.settings.isAutoPublishingEnabled;
```

Comment:
The error seems to be here, look at: https://travis-ci.org/neos/neos-ui/jobs/469367733#L1543